pkg/database/openshiftclusters.go

```diff
@@ -220,6 +221,10 @@ func (c *openShiftClusters) ChangeFeed() cosmosdb.OpenShiftClusterDocumentIterat
 	return c.c.ChangeFeed(nil)
 }
 
+func (c *openShiftClusters) List() (cosmosdb.OpenShiftClusterDocumentIterator, error) {
```

Comment:
should be `func (c *openShiftClusters) List() cosmosdb.OpenShiftClusterDocumentIterator`